### PR TITLE
trivial: Check new plugins for g_print()

### PR DESCRIPTION
**Pull request by hughsie**
This seems to be something that a lot of new submissions get wrong.

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

File: src/fu-usb-backend.c [hunk 1]
```diff
@@ -232,7 +232,7 @@ fu_usb_backend_hotplug_cb(struct libusb_context *ctx,
 	FuUsbBackendIdleHelper *helper;
 	g_autoptr(GMutexLocker) locker = g_mutex_locker_new(&self->idle_events_mutex);
 
-	g_assert(locker != NULL);
+	g_assert(locker != NULL); /* nocheck */
```
Comment (superm1) [inline src/fu-usb-backend.c:235]: shouldn't this be a `g_return_val_if_fail()` too?

**Review by superm1 — COMMENTED** [review]
I guess I'm not sure why we want `g_assert()` anywhere.  Why not go all in on `g_return_val_if_fail` and `g_return_if_fail`?

**Comment by superm1** [comment]
It looks like CI is upset with this, I just stopped the long running job:

```
** (fwupdmgr:422): CRITICAL **: 14:51:36.296: close_nointr: assertion 'fd < 0' failed
(polkitd:391): GLib-GIO-CRITICAL **: 14:51:36.303: g_dbus_method_invocation_get_message: assertion 'G_IS_DBUS_METHOD_INVOCATION (invocation)' failed
(polkitd:391): GLib-GIO-CRITICAL **: 14:51:36.303: g_dbus_message_get_unix_fd_list: assertion 'G_IS_DBUS_MESSAGE (message)' failed
```

**Comment by hughsie** [comment]
>  Why not go all in on g_return_val_if_fail and g_return_if_fail

Good point; we're only using the assert to avoid a compile -Wunused warning. Fixed.

**Review by superm1 — APPROVED** [review]

**Pull request merged**